### DISCUSSION TITLE
Supply surface vs hole open boundary with inlet

### DISCUSSION THREAD

**Discussion by ShehabGamrah** [firemodels/fds#12707] [Q&A] [answered]
I am running simulation of a closed room with an extract surface at one end with 2m3/s and two types of boundary conditions for make-up air. In the 1st run the inlet is a hole with a mesh extended with open boundary. The volume flow rate at both the hole extract are equal and the corridor pressure is around -15 pa. However, in the second run with a supply surface of 2m3/s the volume flow rates plot is similar to the 1st run, but the pressure in this case is overpressurised reaching 1000pa, and increasing. The velocity in both cases is the same. So my question is why with the supply surface the room pressure is increasing however the velocity and volume flow rate values are the same in both cases. Moreover, the supply rate equal extract rate.

**Comment by Er9y714** [comment]
In the first run there are two boundaries. outlet and OPEN, in the second run outlet and inlet. the second run is more likely to be overconstrained by the volume changes (e.g. heating, cooling). For example if there is a heat source in the room, 2 m3/s cold air enters the room, heats up and expands (for example to 3 m3/s) then only 2 m3/s is extracted by the exhaust. The difference of 1 m3/s needs to stay in the room and causes pressure to increase.

**Reply by ShehabGamrah** [reply]
In both cases I used volume flow devices. These values measured 2m3/s for both cases at inlets at outlets. Moreover in both cases there was no heat source only change in parameter was the supply vent vs natural opening. So still don’t know why this change in pressure occurs if volume flow rates are the same.

**Comment by drjfloyd** [comment]
Are your suppply and exhaust at different elevations?  If so the lower vent will have a higher density than the upper vent due to the hydrostatic pressure gradient.

If this isn't the case, submit a simple input file that demonstrates your problem.

**Reply by ShehabGamrah** [reply]
[Test1.zip](https://github.com/firemodels/fds/files/14828963/Test1.zip)
[Test1.zip](https://github.com/firemodels/fds/files/14828964/Test1.zip)
Please find the attached zip file of fds input

**Comment by drjfloyd** [comment]
I said if your vents are not at the same elevation then there will be a density difference. You attached a case with vents at two different heights.  1.1 to 2.1 m is not the same as 1.0 to 2.0 m.  The upper vent will have less mass flow than the lower vent.

&VENT ID='Extraction Vent01', SURF_ID='Extraction02', XB=10.0,10.6,13.3,13.3,1.1,2.1/ 
&VENT ID='Extraction Vent02', SURF_ID='Extraction01', XB=9.7,10.2,5.1,5.1,1.0,2.0/

**Comment by drjfloyd** [comment, marked as answer]
You also have a lower temperature on the supply adding yet more density difference which was a cause noted by Er9y714:

&SURF ID='Extraction01',
      RGB=26,128,26,
      TMP_FRONT=0.0,
      VOLUME_FLOW=-2.0/
